evergreen/evergreen.yml

```diff
@@ -38,32 +38,21 @@ functions:
         shell: bash
         working_dir: mongo-efcore-provider
         script: |
-          # Get the current unique version of this checkout
-          if [ "${is_patch}" = "true" ]; then
-            CURRENT_VERSION=$(git describe)-patch-${version_id}
-          else
-            CURRENT_VERSION=latest
-          fi
-
-          PACKAGE_VERSION=$(bash ./evergreen/generate-version.sh)
+          PACKAGE_VERSION=$(bash triggered_by_git_tag=${triggered_by_git_tag} ./evergreen/generate-version.sh)
 
           export DRIVERS_TOOLS="$(pwd)/../drivers-tools"
           export DOTNET_SDK_PATH="$(pwd)/../.dotnet"
 
           if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
             # Python has cygwin path problems on Windows. Detect prospective mongo-orchestration home directory
             export DRIVERS_TOOLS=$(cygpath -m $DRIVERS_TOOLS)
-          else
-            # non windows OSs don't have dotnet in the PATH
-            export PATH=$PATH:/usr/share/dotnet
```

Comment:
Why is this removed?

Reply:
We simply do not need this code, as we install dotnet into the custom folder under the working folder.